src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -450,6 +450,19 @@
     <Field Name="Method" Type="MethodSymbol"/>
   </Node>
 
+  <!-- Represents the greatest raw metadata token value for any method definition in the current module. -->
```

Comment:
I am not sure what "greatest" is supposed to convey in this comment. Consider elaborating.